egs/commonvoice/asr1/run.sh

```diff
@@ -55,23 +53,33 @@ set -e
 set -u
 set -o pipefail
 
-train_set=valid_train
-train_dev=valid_dev
-recog_set="valid_dev valid_test"
+train_set=valid_train_${lang}
+train_dev=valid_dev_${lang}
+test_set=valid_test_${lang}
+recog_set="valid_dev_${lang} valid_test_${lang}"
 
-if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then
+if [ ${stage} -le -1 ] && [ ${stop_stage} -ge -1 ]; then 
     echo "stage -1: Data Download"
     mkdir -p ${datadir}
-    local/download_and_untar.sh ${datadir} ${data_url}
+    local/download_and_untar.sh ${datadir} ${data_url} ${lang}.tar.gz
 fi
 
 if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
     ### Task dependent. You have to make data the following preparation part by yourself.
     ### But you can utilize Kaldi recipes in most cases 
-    for part in valid-train valid-dev valid-test; do
+    for part in validated_${lang}; do
         # use underscore-separated names in data directories.
-        local/data_prep.pl ${datadir}/ cv-${part} data/"$(echo ${part} | tr - _)"
+        local/data_prep.pl ${datadir} validated data/"$(echo ${part} | tr - _)"
     done
+
+    # Kaldi Version Split
+    # ./utils/subset_data_dir_tr_cv.sh data/validated data/valid_train data/valid_test_dev
+    # ./utils/subset_data_dir_tr_cv.sh --cv-spk-percent 50 data/valid_test_dev data/valid_test data/valid_dev
+
+    # ESPNet Version
+    # consider duplicated sentences
```

Comment:
You can also state that this does not consider the speaker split and this way of split is used for the voxforge recipe.

Reply:
For train-dev-test split,
If we use the official test and dev set and use the rest of the validated set as training set as we discussed this morning, we may encounter the same problem as last week (duplicated word sequence in test and train). That will be a problem for some languages (e.g. welsh) as they will get a very small WER with 10s hours of training data. What do you think about it?

Reply:
how about removing the duplicated word sequence appeared in the dev/test set from the train set?

Reply:
It is a possible solution, I will have a try~ Thanks